system_tests/block_validator_test.go

```diff
@@ -44,10 +45,36 @@ func testBlockValidatorSimple(t *testing.T, dasModeString string, expensiveTx bo
 		chainConfig = params.ArbitrumDevTestDASChainConfig()
 	}
 	_, err = l1NodeConfigA.DataAvailability.Mode()
+
 	Require(t, err)
+
 	l2info, nodeA, l2client, l1info, _, l1client, l1stack := CreateTestNodeOnL1WithConfig(t, ctx, true, l1NodeConfigA, chainConfig)
+
 	defer l1stack.Close()
 
+	usingDas := nodeA.DataAvailService
+
+	if usingDas != nil {
+		keysetBytes, err := usingDas.CurrentKeysetBytes(ctx)
+		Require(t, err)
+		abiBytes := []byte{0xd1, 0xce, 0x8d, 0xa8}
+		var buf [32]byte
+		buf[31] = 0x20
+		abiBytes = append(abiBytes, buf[:]...)
+		buf[30] = byte(len(keysetBytes) / 256)
+		buf[31] = byte(len(keysetBytes) % 256)
+		abiBytes = append(abiBytes, buf[:]...)
+		abiBytes = append(abiBytes, keysetBytes...)
+		for len(abiBytes)%32 != 4 {
+			abiBytes = append(abiBytes, byte(0))
+		}
+		tx := l1info.PrepareTx("RollupOwner", "SequencerInbox", 2000000, big.NewInt(0), abiBytes)
```

Comment:
We should create an abigen wrapper for the sequencer inbox instead of doing manual ABI serialization, as we do in `arbnode/node.go` for the `SetIsBatchPoster` call